frontend/src/entities/group/group.reducer.ts

```diff
@@ -70,6 +70,7 @@ export const addGroupUsers = createAsyncThunk('group/add_users', async (data: Ad
 export const GroupSlice = createSlice({
     name: 'group',
     initialState,
+    reducers: {},
```

Comment:
Can you tell me why this is needed, just for my education.

Reply:
For no other reason than `createSlice` expects a `reducers` field so my linter required _something_ there.